bikeshed/__init__.py

```diff
@@ -1502,7 +1502,7 @@ def idlReplacer(match):
             return E.code({"class":"idl"},
                 E.a({"data-link-type":linkType, "for": match.group(1)}, match.group(2)))
 
-        elementRe = re.compile(r"<{(?:(\w+)/)?(\w+)}>")
+        elementRe = re.compile(r"<{(?:([^ \"'>/=]+)/)?([^ \"'>/=]+)}>")
```

Comment:
This regex is capable of matching across two element defs on the same line, because it's too liberal.  Just do `[\w-]+`; it's a pattern I use a bunch all throughout.